editor.planx.uk/src/@planx/components/Checklist/Editor/OptionsEditor.tsx

```diff
@@ -1,104 +1,46 @@
+import {
+  BaseOptionsEditor,
+  BaseOptionsEditorProps,
+} from "@planx/components/shared/BaseOptionsEditor";
 import React from "react";
-import ImgInput from "ui/editor/ImgInput/ImgInput";
 import SimpleMenu from "ui/editor/SimpleMenu";
-import Input from "ui/shared/Input/Input";
-import InputRow from "ui/shared/InputRow";
-import InputRowItem from "ui/shared/InputRowItem";
 
-import { FlagsSelect } from "../../shared/FlagsSelect";
-import { OptionEditorProps } from "../types";
+export type ChecklistOptionsEditorProps = BaseOptionsEditorProps & {
+  index: number;
+  groupIndex?: number;
+  groups?: Array<string>;
+  onMoveToGroup?: (itemIndex: number, groupIndex: number) => void;
+  showValueField?: boolean;
+};
 
-export const OptionEditor: React.FC<OptionEditorProps> = (props) => {
+const ChecklistOptionsEditor: React.FC<ChecklistOptionsEditorProps> = ({
+  value,
+  onChange,
+  showValueField = false,
+  groups,
+  onMoveToGroup,
+  index,
+}) => {
   return (
-    <div style={{ width: "100%" }}>
-      <InputRow>
-        {props.value.id ? (
-          <input type="hidden" value={props.value.id} readOnly />
-        ) : null}
-        <InputRowItem width="200%">
-          <Input
-            required
-            format="bold"
-            multiline
-            value={props.value.data.text || ""}
-            onChange={(ev) => {
-              props.onChange({
-                ...props.value,
-                data: {
-                  ...props.value.data,
-                  text: ev.target.value,
-                },
-              });
-            }}
-            placeholder="Option"
-          />
-        </InputRowItem>
-
-        <ImgInput
-          img={props.value.data.img}
-          onChange={(img) => {
-            props.onChange({
-              ...props.value,
-              data: {
-                ...props.value.data,
-                img,
-              },
-            });
-          }}
+    <BaseOptionsEditor
+      value={value}
+      onChange={onChange}
+      showValueField={showValueField}
+    >
+      {typeof index !== "undefined" && groups && onMoveToGroup && (
+        <SimpleMenu
```

Comment:
The layout of this is not quite right on the Pizza, the menu has spilled over to the next row - https://4041.planx.pizza/bromley/article4/nodes/_root/nodes/wRCvCPZxVT/edit

![image](https://github.com/user-attachments/assets/6c4810fd-506b-4670-9cdb-68a1c764c1d0)

Reply:
Good spot, thank you! Have just committed a fix for this :)